src/System.Reflection.Metadata/tests/Metadata/Ecma335/Encoding/BlobEncodersTests.cs

```diff
@@ -388,7 +388,350 @@ public void FixedArgumentsEncoder_AddArgument()
         // NamedArgumentTypeEncoder
         // CustomAttributeArrayTypeEncoder
         // CustomAttributeElementTypeEncoder
-        // SignatureTypeEncoder
+
+        [Fact]
+        public void SignatureTypeEncoder_Primitives()
+        {
+            var b = new BlobBuilder();
+            var e = new SignatureTypeEncoder(b);
+            Assert.Same(b, e.Builder);
+
+            e.Boolean();
+            AssertEx.Equal(new byte[] { 0x02 }, b.ToArray());
```

Comment:
I think this is a hold-over from older versions of xunit.

---
In reply to: [63406050](https://github.com/dotnet/corefx/pull/8419#discussion_r63406050) [](ancestors = 63406050)